queries.sql

```diff
@@ -43,3 +43,96 @@ FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+-- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. 
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+
+-- Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+SELECT * FROM animals; -- species column = unspecified
+ROLLBACK;
```

Comment:
- [ ] In accordance to the project requirement, we are to verify if the changes persist after "ROLLBACK", to achieve this I think you can use the `SELECT * FROM animals;` query . 🙏 
- [] Kindly update the screenshot in the PR as stated in the requirement. Thanks 

<img width="715" alt="Untitled" src="https://github.com/Kingno3l/Vet-Clinic/assets/103386183/75ffd493-33cb-444b-b5ae-0d3112800c95">

 
```suggestion
SELECT * FROM animals; -- species column = unspecified
ROLLBACK;
SELECT * FROM animals;
```

Reply:
The screenshot is there. I forgot to add SELECT * FROM animals; in my queries ..Thanks👍